bindings/experimental/distrdf/python/DistRDF/Backends/Base.py

```diff
@@ -159,7 +159,16 @@ def mapper(current_range):
                 list: This respresents the list of (mergeable)values of all
                 action nodes in the computational graph.
             """
-            import ROOT
```

Comment:
Was this a redundant import ROOT? Can't see it in this commit.

Reply:
Yes it was a redundant import since now the ROOT facade can be properly serialized/deserialized. We still haven't removed it but since I'm touching both the mapper and reducer functions in this PR I also removed those imports. I can add them back and remove them in another PR if it's preferable

Reply:
It's ok it can stay here.